Tests/UnitTests/Core/Vertexing/IterativeVertexFinderTests.cpp

```diff
@@ -25,6 +26,7 @@
 #include "Acts/Vertexing/IterativeVertexFinder.hpp"
 #include "Acts/Vertexing/Vertex.hpp"
 #include "Acts/Vertexing/VertexFinderConcept.hpp"
+#include "VertexingDataHelper.cpp"
```

Comment:
Splitting into ```.cpp``` and header not possible, since the cpp is not compiled in the unit tests. I moved everything to a header and made the function ```inline```.